README.md

```diff
@@ -12,4 +12,7 @@ Page content type for Tide distribution
 
 # Caveats
 
-Tide Page is on the alpha release, use with caution. APIs are likely to change before the stable version, that there will be breaking changes and that we're not supporting it for external production sites at the moment.
\ No newline at end of file
+Tide Page is on the alpha release, use with caution. APIs are likely to change before the stable version, that there will be breaking changes and that we're not supporting it for external production sites at the moment.
```

Comment:
I think it's time to change this language. This is stable and unlikely to change much from now.

Reply:
This is outside of the scope of this PR. Let's not block it for the readme stuff.